doc/Alerting/Rules.md

```diff
@@ -28,39 +28,44 @@ __Conditions__ can be any of:
 - Smaller or Equal `<=`
 
 __Values__ can be an entity or any single-quoted data.
-__Glues__ can be either `&&` for `AND`.  or `||` for `OR`. Note if you need to use `OR` `||` please use a macro. 
+__Glues__ can be either `&&` for `AND`.  or `||` for `OR`. 
+
+**Note** if you need to use `OR` `||` please use a [macros](Macros.md)
 
 __Note__: The difference between `Equals` and `Like` (and its negation) is that `Equals` does a strict comparison and `Like` allows the usage of MySQL RegExp.
 
 Arithmetics are allowed as well.
 
 # Options
+
 Here are some of the other options available when adding an alerting rule:
 
-- Rule name: The name associated with the rule.
-- Severity: How "important" the rule is.
-- Max alerts: The maximum number of alerts sent for the event.  `-1` means unlimited.
-- Delay: The amount of time in seconds to wait after a rule is matched before sending an alert.
-- Interval: The interval of time in seconds between alerts for an event until Max is reached.
-- Mute alerts: Disable sending alerts for this rule.
-- Invert match: Invert the matching rule (ie. alert on items that _don't_ match the rule).
++ Rule name: The name associated with the rule.
```

Comment:
Any reason these have been changed to +? It does the same thing but `-` are more common for lists imho.

Reply:
I made that change because the - was not working and I found that the + did work. It was wonky if you look at the doc. idk maybe formatting issue that im not seeing. 

![image](https://user-images.githubusercontent.com/27959419/33187898-eda321b6-d05a-11e7-94e9-94575b134783.png)

Reply:
It possibly just needs a new line below Alert when: as it should be fine.

Reply:
Yes, you must have a new line before a list.